src/SFA.DAS.AssessorService.Application/Handlers/Staff/StartCertificateHandler.cs

```diff
@@ -55,6 +55,8 @@ public async Task<Certificate> Handle(StartCertificateRequest request, Cancellat
                     certData.LearningStartDate = ilr.LearnStartDate;
                     certData.FullName = $"{ilr.GivenNames} {ilr.FamilyName}";
                     certificate.CertificateData = JsonConvert.SerializeObject(certData);
+                    
+                    // To be updated based on ILR Funding Model
```

Comment:
When will it be updated?

Reply:
I was attempting to keep this PR as just the removal of privately funded without adding new stuff in to try and keep the commits clean. The SV-456 branch it's going into is our Epic branch for developing SV Record an Outcome so it will be updated there with another PR before merging into master.

Reply:
I've updated it here so it's not lost, it's a simple change.